test/stubs/repositories-config/expected.json

```diff
@@ -0,0 +1,24 @@
+{
+  "package": {
+    "dependencies": {
+      "Moq": {
+        "dependencies": {},
```

Comment:
no transitives?

Reply:
not for the example they provided. I can create one with.